201-cdn-with-ruleseengine-urlsigning/README.md

```diff
@@ -18,3 +18,10 @@
 
 This template creates a CDN Profile and a CDN Endpoint with a user specified origin and all of our most commonly used settings on CDN. This template also configures rules engine UrlSigning action for default and override parameters.
 
+Before using the template, make sure to do the following:
+
+1. Create a key vault in the same subscription or in a different subscription.
+2. Define two secrets and set up secret values manually (at least 32 size)
+3. References the two secrets in urlSigningKeys of the template (change subscriptionId, resourceGroupName, vaultName, secretName, secretVersion)
+4. Make sure the keyvault is given "Get Permissions for Secret" access to "Microsoft.Azure.CDN" Principal.
```

Comment:
how do I do this?  the principal does not exist in the tenant...

Reply:
@bmoore-msft , Please see below. 
![image](https://user-images.githubusercontent.com/56672327/88721104-c54fa300-d0da-11ea-84a6-5f77b6b557c0.png)

Reply:
the principal does not exist in the tenant...

Reply:
@bmoore-msft , what tenant are you using? I checked with both COPR and AME tenant. I could see the principal.

Reply:
You may need to register if it doesn't exist. https://docs.microsoft.com/en-us/azure/cdn/cdn-custom-ssl?tabs=option-2-enable-https-with-your-own-certificate

![image](https://user-images.githubusercontent.com/56672327/89081334-fd0b5480-d33f-11ea-9a55-e3c19931a0c4.png)

Reply:
@bmoore-msft , can you please check if all looks good for you?

Reply:
let's add this step to the readme... so others don't get hung up.  It looks like the SP is given contributor perms to the sub by default so I suspect that step will always be required.

Everything else looks good.